wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/CreateGroupConversationActionHandler.swift

```diff
@@ -140,6 +141,44 @@ final class CreateGroupConversationActionHandler: ActionHandler<CreateGroupConve
         case (403, "access-denied"):
             action.fail(with: .accessDenied)
 
+        case (409, _):
```

Comment:
Is it possible to check the payload label similarly to the other cases? 
Such as below
```suggestion
        case (409, "non-federating-backends"):
```

Reply:
No, we can't. `non-federating-backends` is not a label